doc/cephfs/quota.rst

```diff
@@ -45,15 +45,28 @@ To view quota limit::
    system call. Instead, a specific CephFS extended attribute can be viewed by
    running ``getfattr /some/dir -n ceph.<some-xattr>``.
 
-To remove a quota, set the value of extended attribute to ``0``::
+To remove a quota, remove the respective extended attribute or set the value to ``0``.
+Setting the value to ``0``, disables quota for the inode.
```

Comment:
Ah, this doc change could be clearer regarding quota-- will clarify.

Regarding `ceph.dir.layout`, if you are referring to the behavior in the other comment here: https://github.com/ceph/ceph/pull/53755#discussion_r1483180400, IIUC, there is no difference.

Reply:
There is a difference: if a directory sets the layout explicitly to the default layout (or any layout), then all new files inherit that layout unless overridden. Consider:

```
setfattr -n ceph.dir.layout -v <nondefault_layout> /one
setfattr -n ceph.dir.layout -v <default_layout> /one/two
touch /one/two/three
```

File `/one/two/three` will inherit `<default_layout>`  instead of `<nondefault_layout>`. The way to get it to inherit `<nondefault_layout>` is to **remove** the layout on `/one/two`.